### PR TITLE
feat: update email templates

### PR DESCRIPTION
**Pull request by dguyen**
Changes:

- Improve consistency across email templates
- Improve button alignments
- Add SPM email attachment
- Add "Sign In" button for forgot password email template
- Update SPM email template to promote new accounts

### CODE REVIEW

File: apps/marketing/src/components/(marketing)/single-player-mode/create-single-player-document.action.ts [hunk 1]
```diff
@@ -177,9 +177,7 @@ export const createSinglePlayerDocument = async (
     },
   );
 
-  // Todo: Handle `downloadLink`
   const template = createElement(DocumentSelfSignedEmailTemplate, {
-    downloadLink: `${process.env.NEXT_PUBLIC_MARKETING_URL}/single-player-mode/${documentToken}`,
```
Comment (Mythie) [inline apps/marketing/src/components/(marketing)/single-player-mode/create-single-player-document.action.ts:182]: We still want the download link I think 🤔

Comment (dguyen) [reply apps/marketing/src/components/(marketing)/single-player-mode/create-single-player-document.action.ts:182]: How come? The "download" link just takes them back to the SPM success page

The PDF is now directly attached to the email, so they can directly view/download it

Comment (Mythie) [reply apps/marketing/src/components/(marketing)/single-player-mode/create-single-player-document.action.ts:182]: Yep you're right I'm the silly one

File: packages/email/template-components/template-document-self-signed.tsx [hunk 2]
```diff
@@ -49,7 +49,7 @@ export const TemplateDocumentSelfSigned = ({
         <Text className="mx-auto mb-6 mt-1 max-w-[80%] text-center text-base text-slate-400">
           Create a{' '}
           <Link
-            href={`${process.env.NEXT_PUBLIC_WEBAPP_URL}/signup`}
+            href={`${process.env.NEXT_PUBLIC_WEBAPP_URL ?? 'http://localhost:3000'}/signup`}
```
Comment (Dun-sin) [inline packages/email/template-components/template-document-self-signed.tsx:52]: don't you think this will be neater doing it in a variable?

Comment (Dun-sin) [reply packages/email/template-components/template-document-self-signed.tsx:52]: of cuz it has nothing to do with your PR, but could be a good first issue

Comment (dguyen) [reply packages/email/template-components/template-document-self-signed.tsx]: Hi Dun-sin!

Thanks for pointing that out, it made me realise that this path is also required somewhere else in this file.

I'll update it as part of this PR, thanks!

Comment (Dun-sin) [reply packages/email/template-components/template-document-self-signed.tsx]: I noticed it's a universal thing in some other files

**Pull request merged**